server/modules/user/api/userApi.js

```diff
@@ -60,6 +60,17 @@ module.exports.init = app => {
 
   // ==== UPDATE
 
+  app.put('/user/username', async (req, res, next) => {
+    try {
+      const { user } = req
+      const { name } = Request.getParams(req)
+
+      res.json(UserService.updateUsername(user, name))
```

Comment:
don't you need to await here? would it be better to have a variable?

Reply:
Added await